lib/index.js

```diff
@@ -349,6 +349,25 @@ api.createIndexes = (options, callback) => async.each(
   options, (item, callback) => api.collections[item.collection].createIndex(
     item.fields, item.options, callback), callback);
 
+/**
+ * Creates a streaming GridFS bucket interface.
+ *
+ * By default the writeOptions config value is used for the GridFSBucket
+ * writeConcern option.
+ *
+ * @param options see GridFSBucket documentation
+ * @param callback(err, bucket) called once the operation completes.
+ */
+api.createGridFSBucket = (options, callback) => {
+  let opts = options;
+  if(!('writeConcern' in options)) {
```

Comment:
could this use `_.defaults` and avoid all the opts/options gyrations?  Although `lodash` is not required in this file, it's already a dependency.

Reply:
I was micro optimizing for no big reason.  Will simplify.